core/src/test/resources/valid_iam_token.json

```diff
@@ -0,0 +1,7 @@
+{
+  "access_token": "aaaa-1111",
+  "refresh_token": "99999999",
+  "token_type": "Bearer",
+  "expires_in": 3600,
+  "expiration": 999999999999999999
```

Comment:
If someone is still maintaining this thousands of years from now, I hope they remember to change this expiration time!